monkey/infection_monkey/post_breach/shell_startup_files/windows/shell_startup_files_modification.py

```diff
@@ -0,0 +1,12 @@
+SHELL_STARTUP_FILE = '$Profile'
+
+
+def get_windows_commands_to_modify_shell_startup_files():
```

Comment:
Why are we doing this through powershell commands rather than opening the file in python?

Reply:
I guess this was taken from atomic red team, where scripts are in powershell

Reply:
Should I change it to open it in python itself?